ci-test.sh

```diff
@@ -18,15 +18,15 @@ core)
     # A 64bit little-endian and a 32bit big-endian target.
     # (Varying the OS is totally pointless for core.)
     for TARGET in x86_64-unknown-linux-gnu mips-unknown-linux-gnu; do
-        echo "::group::Testing core ($TARGET, no validation, no Stacked Borrows, symbolic alignment)"
+        echo "::group::Testing coretests ($TARGET, no validation, no Stacked Borrows, symbolic alignment)"
```

Comment:
```suggestion
        # There are no library tests in core, and the integration tests are in a separate crate.
        echo "::group::Testing coretests ($TARGET, no validation, no Stacked Borrows, symbolic alignment)"
```
That's correct, right?

Reply:
Indeed.